packages/cli/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @shopify/cli-hydrogen
 
+## 5.0.0
+
+### Patch Changes
```

Comment:
Hmm yeah, it's just for the virtual-routes. It looks like 2 of them can be made devDependencies because they are just used for types. However, `@remix-run/react` should probably be there since it's used for things like `useLoaderData` 🤔